RecoTracker/TransientTrackingRecHit/python/TTRHBuilderWithTemplate_cfi.py

```diff
@@ -9,9 +9,13 @@
 )
 
 from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
-trackingPhase2PU140.toModify(TTRHBuilderAngleAndTemplate, Phase2StripCPE = cms.string('Phase2StripCPE'))
+trackingPhase2PU140.toModify(TTRHBuilderAngleAndTemplate, Phase2StripCPE = 'Phase2StripCPE')
```

Comment:
@duartej sorry, I realize only now that this parameter is not defined yet (above) so the removal of the cms type is not appropriate here. I expect the tests to fail. Can you revert it (only for this line?)

Reply:
sure @mmusich no worries,  done.